src/index.js

```diff
@@ -83,6 +84,8 @@ function ParseServer({
   enableAnonymousUsers = true,
   oauth = {},
   serverURL = '',
+  verifyUserEmails = false,
```

Comment:
Suggested name: `userEmailVerificationEnabled`, though I come from the land of long and weird names and your's might be better.
Also, maybe `enableUserEmailVerification`, since we have `enableAnonymousUsers`.

The very first one is prefferred though.